haskell-indexer-plugin/src/Haskell/Indexer/Plugin.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Haskell.Indexer.Plugin where
+
+import GhcPlugins hiding (($$))
+import TcRnTypes
+
+import TcRnMonad
+import TcRnDriver
+import GhcMonad
+import Data.IORef
+
+import Language.Haskell.Indexer.Backend.Ghc
+import Language.Haskell.Indexer.Backend.GhcEnv
+import Language.Haskell.Indexer.Translate
+import Language.Haskell.Indexer.Args
+
+import System.IO
+import System.FilePath
+import System.Directory
+
+plugin :: Plugin
```

Comment:
Could add a (module) comment pointing to upstream source plugin docs / other useful resource.